.github/workflows/triggers-functions-offline-testing.yml

```diff
@@ -0,0 +1,96 @@
+---
+name: triggers-offline-testing
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+env:
+  TRIGGERS_GETTING_STARTED_FOLDER: functions/triggers-getting-started
+  OFFLINE_SERVER_URL: http://localhost:8080
+
+jobs:
+  trigger-node:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Install node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 20
+
+      - name: Install node dev requirements
+        run: |
+          cd ${{ env.TRIGGERS_GETTING_STARTED_FOLDER }}/node
+          npm install
+
+      - name: Install python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.11"
+
+      - name: Install testing requirements
+        run: pip install -r ${{ env.TRIGGERS_GETTING_STARTED_FOLDER }}/tests/requirements.txt
+
+      - name: Run node offline testing
+        uses: BerniWittmann/background-server-action@v1
+        with:
+          # The wait-on directive doesn't work with 405 on / with GETs
```

Comment:
On other examples, with use `wait-on` to wait for the server to start before running the tests, but this doesn't work if the server returns a 405 on gets /